### ISSUE TITLE
please im 0 at source help me

### ISSUE THREAD

**Issue by NovaPlay1** [DuskieWhy/NightmareVision#17]
### Describe your problem here.

i managed to download all things but this still dont working nhagaggg😭
![image](https://github.com/user-attachments/assets/55b0570f-fd4e-4749-bf14-f81ef511f06b)

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

no

**Comment by NovaPlay1** [comment]
why the f its help wanted

**Comment by moxie-coder** [comment]
> why the f its help wanted

you chose the wrong template, and you need haxe 4.3

**Comment by NovaPlay1** [comment]
okaay, i will check later

**Comment by NovaPlay1** [comment]
i have haxe 4.3.6 still not working

**Comment by Snowy-0** [comment]
Yeah, I'm getting the same issue here. I have Haxe 4.3.0 and it gives the same error. Even tried reinstalling it and it's libraries. I went into the file myself and added the ; myself but it for some reason gives the same error? Not sure why.
